<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainApp</class>
 <widget class="QWidget" name="MainApp">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>160</width>
     <height>421</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QPushButton" name="buttonLogo">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>나눔바른고딕</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border-image: url(:/resources/images/logo.png);
    background-repeat: no-repeat;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonBroadcastAll">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>1</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #1C75BC; 
	color: white;
	border-style: solid; 
	border-width: 0px; 
	border-color: #1C75BC;
}</string>
      </property>
      <property name="text">
       <string>전체방송</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonBroadcastGrp">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>1</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #BAA868; 
	color: white; 
	border-style: solid; 
	border-width: 0px; 
	border-color: #BAA868;
}</string>
      </property>
      <property name="text">
       <string>그룹방송</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonBroadcastIndividual">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>1</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #C75252; 
	color: white; 
	border-style: solid; 
	border-width: 0px; 
	border-color: #C75252;
}</string>
      </property>
      <property name="text">
       <string>개별방송</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="ScrollingLabel" name="labelNews">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>420</y>
     <width>801</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color :#005077; color : white; padding: 0px 0px 0px 0px;</string>
   </property>
   <property name="text">
    <string>11월 13일부터 거리두기 단계별 위험시설에서 마스크 미착용 시 과태료가 부과됩니다. 모두의 안전을 위해 외출시 마스크를 꼭 착용해주시기 바랍니다.</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="ViewControl">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>0</y>
     <width>641</width>
     <height>421</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">QStackedWidget {
	background-color: #160C4B; 
	border-style: solid; 
	border-width: 0px; 
	border-color: #160C4B;
}</string>
   </property>
   <widget class="TimeDisplay" name="page">
    <property name="styleSheet">
     <string notr="true">
	background-color: #160C4B; 
	border-style: solid; 
	border-width: 0px; 
	border-color: #160C4B;
</string>
    </property>
   </widget>
   <widget class="BroadCastDisplay" name="WidgetBroadcast">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">
	background-color: #160C4B; 
	border-style: solid; 
	border-width: 0px; 
	border-color: #160C4B;
</string>
    </property>
   </widget>
   <widget class="GroupDisplay" name="WidgetGroup">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">
	background-color: #160C4B; 
	border-style: solid; 
	border-width: 0px; 
	border-color: #160C4B;
</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScrollingLabel</class>
   <extends>QLabel</extends>
   <header>scrollinglabel.h</header>
  </customwidget>
  <customwidget>
   <class>TimeDisplay</class>
   <extends>QWidget</extends>
   <header>timedisplay.h</header>
  </customwidget>
  <customwidget>
   <class>BroadCastDisplay</class>
   <extends>QWidget</extends>
   <header>broadcastdisplay.h</header>
  </customwidget>
  <customwidget>
   <class>GroupDisplay</class>
   <extends>QWidget</extends>
   <header>groupdisplay.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
